### PR TITLE
Milestone two

### PR DESCRIPTION
**Pull request by helciodev**
This pull request adds: folders for components, reducers, action, and containers, respective files, and initializes the redux store.

### CODE REVIEW

File: src/containers/BooksList.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from '../components/Book';
+
+function BooksList() {
+  const books = useSelector((state) => state.booksReducer);
```
Comment (taiwo2) [inline src/containers/BooksList.js:6]: As specified by the projects requirement  it uses `mapStateToProps` to fetch books from the `store`

File: src/reducers/books.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { REMOVE_BOOK, CREATE_BOOK } from '../actions/index';
+
+const initialState = {
+  books: [],
+};
+
+function booksReducer(state = initialState, action) {
+  if (action.type === REMOVE_BOOK) { /* */ }
+
+  if (action.type === CREATE_BOOK) { /* */ }
```
Comment (taiwo2) [inline src/reducers/books.js]: should return the states object

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@helciodev congratulations on this milestone almost all the project requirement is met, however, few changes are required, and kindly find my inline comments

### Highlights
Git flow  👍🏻 
project structure 👍🏻 
### improve

- [ ] has children components wrapped in `<div>` tags
- [ ] it uses `mapStateToProps` to fetch books from the `store`
- [ ] it implements CREATE_BOOK and REMOVE_BOOK actions 

### once specific changes are made kindly request for review

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BooksForm from '../containers/BooksForm';
+import BooksList from '../containers/BooksList';
+
+function App() {
+  return (
+    <>
+      <BooksList />
+      <BooksForm />
```
Comment (taiwo2) [inline src/components/App.js:9]: should be wrapped in `div` tag

File: src/containers/BooksList.js (hunk 1 above)
Comment (helciodev) [reply src/containers/BooksList.js:6]: Hi there, Taiwo in the requirements for this project states that I can either fetch the books from the state by using connect or redux hooks I chose the latter.

**Review by george-shammar — APPROVED** [review]
Hi @helciodev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**